### DISCUSSION TITLE
RcTrainer function under MICROPYTHON

### DISCUSSION THREAD

**Discussion by toadzhou** [micropython/micropython#10191] [Using MicroPython]
If ESP32 implements the function of RcTrainer under MICROPYTHON？
Arduino-like RcTrainer library http://www.airspayce.com/mikem/arduino/RcTrainer/

**Comment by toadzhou** [comment]
I want to use ESP32 to read the 3.5MM PPM signal on the RC remote control

**Reply by jimmo** [reply]
I suspect this can be done, but I do not know of any implementations for ESP32. See https://forum.micropython.org/viewtopic.php?t=1163 for an STM32 version.
